.github/workflows/workflow.yml

```diff
@@ -166,50 +176,65 @@ jobs:
         sudo apt-get update --fix-missing
         sudo apt-get autoremove
         sudo apt-get autoclean
-        pip install pipenv
         pip install tox
         sudo apt-get install -y protobuf-compiler
     # use sudo rm /var/lib/apt/lists/lock above in line above update if dependency install failures persist
     # use sudo apt-get dist-upgrade above in line below update if dependency install failures persist
     - if: matrix.os == 'macos-latest'
       name: Install dependencies (macos-latest)
       run: |
-        pip install pipenv
         pip install tox
         brew install protobuf
     - if: matrix.os == 'windows-latest'
       name: Install dependencies (windows-latest)
       run: |
-        pip install pipenv
         pip install tox
         echo "::add-path::C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x64"
         choco install protoc
         python scripts/update_symlinks_cross_platform.py
     - name: Unit tests and coverage
       run: |
         tox -e py${{ matrix.python-version }} -- -m 'not integration and not unstable'
-    # optionally, for all tests, remove 'not unstable' to run unstable tests as well
-    - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v1
+
+  platform_checks_sync_aea_loop:
+    continue-on-error: True
+    needs:
+    - common_checks_1
+    - common_checks_2
+    - common_checks_3
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions/setup-python@master
       with:
-        token: ${{ secrets.CODECOV_TOKEN }}
-        file: ./coverage.xml
-        flags: unittests
-        name: codecov-umbrella
-        yml: ./codecov.yml
-        fail_ci_if_error: false
-        
+        python-version: 3.8
+    - uses: actions/setup-go@master
+      with:
+        go-version: '^1.14.0'
+    - name: Install dependencies (ubuntu-latest)
+      run: |
+        sudo apt-get update --fix-missing
+        sudo apt-get autoremove
+        sudo apt-get autoclean
+        pip install tox
+        sudo apt-get install -y protobuf-compiler
+    - name: Unit tests and coverage
+      run: |
+        tox -e py3.8 -- --aea-loop sync -m 'not integration and not unstable'
+
   golang_checks:
+    continue-on-error: false
```

Comment:
was it wanted? The other jobs have `true`

Reply:
I will change